core/assert.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+#include "assert.h"
+
+#ifdef DEBUG_ASSERT_VERBOSE
+NORETURN void _assert_failure(const _assert_failure_info_t *info)
+{
+    printf("%s:%d => ", info->file, info->line); \
```

Comment:
Unless there is a int2str function you know of that would be available without requiring to pull in `fmt` everytime assert is used, I'm not sure how to do this apart from using "sprintf()" (which would also be a printf).